### ISSUE TITLE
Question: Can we assign weights_per_class for Arcface Loss?

### ISSUE THREAD

**Issue by TEnsorTHiru** [KevinMusgrave/pytorch-metric-learning#532]
I have been using Arcface Loss function with Efficientnetb3 model with a dataset of 22 categories. Several category has 15000 samples but there are nine classes with very low number of samples. Is it any way to include loss weighting for these classes with ArcFace. Or else Can i use anyother loss function that might help just like arcface does.

**Comment by TEnsorTHiru** [comment]
Thanks again for the help
